packages/rich-text-types/src/nodeTypes.ts

```diff
@@ -94,6 +94,14 @@ export interface Link<T extends string = string> {
   };
 }
 
+export interface ResourceLink {
+  sys: {
+    type: 'ResourceLink';
+    linkType: 'Contentful:Entry';
+    urn: string;
+  };
+}
```

Comment:
our schema for third-party resources might differ slightly from this, but TBD. We'll want to make sure this interface supports both external Contentful and external 3P resources, for sure!

Reply:
We can turn this into a type union in the future. Right now I don't see a blocker.

Reply:
We are trying to create alignment on the topic of how to shape those 3rd party links outside of this PR. After that we will make sure to amend the schema asap.